test/test-parse-class.cpp

```diff
@@ -212,6 +212,18 @@ TEST(test_parse, class_statement_with_methods) {
                 ElementsAre(spy_visitor::visited_property_declaration{u8"m"}));
   }
 
+  {
```

Comment:
I'd put this test into its own `TEST`. All other tests in this `TEST` assert no error.